build.sh

```diff
@@ -138,12 +141,19 @@ build_v8() {
 	# The build actually requires that you clone using this git wrapper tool
 	cd ${wrkdir}
 	OLDPATH=${PATH}
-	PATH=${wrkdir}/depot_tools:${PATH}
-	fetch v8 || exit $?
+	# v8's build needs python 2.7.10; as we've already built that, we might
+	# as well use it rather than forcing the user to install their own.
+	PATH=${wrkdir}/cpython-inst/bin:${wrkdir}/depot_tools:${PATH}
+	# XXX we should check for errors when fetching, but currently that
+	# causes problems because fetch runs a script which aborts on OpenBSD
```

Comment:
Ah, so v8 doesn't actually build? Or you need to run the build twice to get it to build? Or?